doc/man1/openssl-enc.pod

```diff
@@ -58,10 +59,14 @@ either by itself or in addition to the encryption or decryption.
 
 Print out a usage message.
 
-=item B<-ciphers>
+=item B<-list>
 
 List all supported ciphers.
 
+=item B<-ciphers>
+
+Alias of -list to display all supported ciphers.
+
```

Comment:
The only essential change I made while resolving merge conflicts was to place the new favoured option `-list` before the deprecated alias `-ciphers`.